.ci/scripts/install-tools.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+install() {
```

Comment:
@mw-ding do we need a specific version of YARN or NodeJS to build the project?

Reply:
No, AFAIK. you can move ahead with this. I will ping @spacedragon to take a look at this.